src/bindings/python/tests_compatibility/conftest.py

```diff
@@ -44,7 +44,7 @@ def pytest_addoption(parser):
     parser.addoption(
         "--backend",
         default="CPU",
-        choices=["CPU", "GPU", "GNA", "HETERO", "TEMPLATE"],
```

Comment:
please, avoid changes in legacy components, which are being removed in parallel other PRs